server/util/log/log.go

```diff
@@ -129,7 +129,7 @@ func init() {
 }
 
 func LocalLogger() zerolog.Logger {
-	output := zerolog.ConsoleWriter{Out: os.Stdout}
+	output := zerolog.ConsoleWriter{Out: os.Stderr}
```

Comment:
The invocation logs for each action run by the CI runner (the "outer" invocations) shouldn't be affected since we have our own logging interface which forwards both stdout and stderr of each bazel command to a combined stdout stream. (All the invocation logs get sent to stdout).

The CI runner action itself (which doesn't have an invocation associated with it) might be affected though -- we don't surface those logs anywhere but we might want to at some point. To future-proof it, you could change `os.Stdout` to `os.Stderr` here: https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/cmd/ci_runner/main.go#L249

Reply:
Thanks! Updated CI runner to send output to stderr.